meshroom/ui/qml/main.qml

```diff
@@ -902,6 +902,81 @@ ApplicationWindow {
+                                        disableAutoRefresh.checked = false
+                                        minimalAutoRefresh.checked = false
+                                        _reconstruction.filePollerRefreshChanged(0)
+                                    }
+                                    // Prevents cases where the user unchecks the currently checked option
+                                    enableAutoRefresh.checked = true
+                                    filePollerRefreshStatus.text = MaterialIcons.sync
+                                }
+                            }
+                            MenuItem {
+                                id: disableAutoRefresh
+                                text: "Disable File Watcher"
+                                checkable: true
+                                checked: _reconstruction.filePollerRefresh == 1
+                                ToolTip.text: "No file status will be checked"
+                                ToolTip.visible: hovered
+                                ToolTip.delay: 200
+                                onToggled: {
+                                    if (checked) {
+                                        enableAutoRefresh.checked = false
+                                        minimalAutoRefresh.checked = false
+                                        _reconstruction.filePollerRefreshChanged(1)
+                                    }
+                                    // Prevents cases where the user unchecks the currently checked option
+                                    disableAutoRefresh.checked = true
+                                    filePollerRefreshStatus.text = MaterialIcons.sync_disabled
+                                }
+                            }
+                            MenuItem {
+                                id: minimalAutoRefresh
+                                text: "Enable Minimal Auto-Refresh"
+                                checkable: true
+                                checked: _reconstruction.filePollerRefresh == 2
+                                ToolTip.text: "Check the file status of submitted or running chunks periodically"
+                                ToolTip.visible: hovered
+                                ToolTip.delay: 200
+                                onToggled: {
+                                    if (checked) {
+                                        disableAutoRefresh.checked = false
+                                        enableAutoRefresh.checked = false
+                                        _reconstruction.filePollerRefreshChanged(2)
+                                    }
+                                    // Prevents cases where the user unchecks the currently checked option
+                                    minimalAutoRefresh.checked = true
+                                    filePollerRefreshStatus.text = MaterialIcons.sync
```

Comment:
Maybe https://materialui.co/material-icons-two-tone/published-with-changes to have 1 logo per mode?

Reply:
Done.